index.html

```diff
@@ -6066,35 +6066,17 @@ <h2 id="AudioWorkletNode">
               </p>
             </dd>
             <dt>
-              attribute EventHandler ondata
+              readonly attribute MessagePort port
             </dt>
             <dd>
               <p>
-                The <code>ondata</code> handler is called whenever the
-                associated <a>AudioWorkletProcessor</a> sends data back to the
-                <a>AudioWorkletNode</a> instance.
+                Every <a>AudioWorkletNode</a> has an associated
+                <code>port</code> which is a <a href=
+                "https://html.spec.whatwg.org/multipage/comms.html#message-ports">
+                MessagePort</a>. It is connected to the port on the
+                corresponding <a>AudioWorkletProcessor</a> object.
```

Comment:
Suggest a little additional text here and on the other `MessagePort` stating that this is the way that Web Audio allows the AWN and AWP to send data to each other. Technically this is redundant but we could go a little further than just saying that the ports are connected.